README.md

```diff
@@ -56,9 +57,10 @@ Create a `smoke-framework-codegen.json` file in the directory you have created w
 
 This JSON file can contain the following fields-
 * **modelFilePath**: Specifies the absolute or relative (to the base directory path) file path to the Swagger model. Required.
+* **modelFormat**: Specifies the format of the provided model. `OPENAPI3_0` indicates Open API 3.0 and `SWAGGER` indicates a Swagger 2.0 model. Optional, defaults to `Swagger`.
 * **baseName**: A base name for your service (without the "Service" postfix). Required.
 * **applicationSuffix**: The suffix that is combined with the `baseName` to create the service's executable name. Defaults to `Service`.
-* **generationType**: `server` to generate a new service; `serverUpdate` to preserve changes to existing operation handlers. Required.
+* **generationType**: `server` to generate a new service; `serverUpdateWithPlugin` to preserve changes to existing operation handlers while using this generator during the build process to generate model and client files. Required.
```

Comment:
I personally found this sentence hard to understand. Could be just me. Perhaps, this can be written as:

`serverUpdateWithPlugin` to preserve changes to existing operation handlers and regenerate model and client files.